### PR TITLE
DAOTable: add viewMoreMenuItem and handler code --> development

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/foam/dashboard/view/DAOTable.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ foam.CLASS({
   ],
   imports: [
     'dashboardController',
-    'stack'
+    'stack',
+    'pushMenu'
```
Comment (sarthak-marwaha) [inline src/foam/dashboard/view/DAOTable.js]: Sort

Comment (ghost) [reply src/foam/dashboard/view/DAOTable.js]: done.

**Review by sarthak-marwaha — APPROVED** [review]

File: src/foam/dashboard/view/DAOTable.js [hunk 2]
```diff
@@ -171,12 +180,17 @@ foam.CLASS({
       name: 'viewMoreAction',
       label: 'View more activities',
       code: function() {
-        this.stack.push(this.StackBlock.create({
-          view: {
-            class: this.DAOBrowseControllerView,
-            data: this.dao.where(this.predicate),
-          }, parent: this.__subContext__
-        }));
+        if ( ! (this.viewMoreMenuItem === undefined ||
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DAOTable.js]: Is this the same as just: if ( this.viewMoreMenuItem ) ?

File: src/foam/dashboard/view/DAOTable.js [hunk 3]
```diff
@@ -171,12 +180,17 @@ foam.CLASS({
...
+                this.viewMoreMenuItem === '') ) {
+          this.pushMenu(this.viewMoreMenuItem);
+        } else {
+          this.stack.push(this.StackBlock.create({
+            view: {
+              class: this.DAOBrowseControllerView,
+              data: this.dao.where(this.predicate),
+            }, parent: this.__subContext__
```
Comment (kgrgreer) [inline src/foam/dashboard/view/DAOTable.js:190]: Can you just use 'this' here?

Comment (ghost) [reply src/foam/dashboard/view/DAOTable.js:190]: the original code did this. i left it alone

File: src/foam/dashboard/view/DAOTable.js (hunk 2 above)
Comment (ghost) [reply src/foam/dashboard/view/DAOTable.js]: yeah, tested out in the js console, will change this.

**Pull request merged**